qa/tasks/cephfs/test_misc.py

```diff
@@ -504,6 +504,23 @@ def test_dump_inmemory_log_on_missed_internal_heartbeats(self):
                 return
         self.assertTrue(False, "Failed to dump in-memory logs during missed internal heartbeat")
 
+    def _session_client_ls(self, cmd: list):
+        info = self.fs.mds_asok(cmd)
+        self.assertIsNotNone(info)
+        mount_a_client_id = self.mount_a.get_global_id()
+        info = self.fs.mds_asok(cmd + [f"id={mount_a_client_id}"])
+        self.assertEqual(len(info), 1)
+        self.assertEqual(info[0]["id"], mount_a_client_id)
+        info = self.fs.mds_asok(cmd + ['--cap_dump'])
+        for i in range(0, len(info)):
+            self.assertIn("caps", info[i])
+
+    def test_session_ls(self):
+        self._session_client_ls(['session', 'ls'])
```

Comment:
Verify the mount is in the session listing.

Reply:
Added assert for `mount_point`